File: src/main/java/net/eduard/permissions/command/PermissionsGroupRemovePermissionCommand.java

package net.eduard.permissions.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;
import net.eduard.permissions.EduPermissions;
import net.eduard.permissions.manager.PermissionsGroup;
import net.eduard.permissions.manager.PermissionsManager;

public class PermissionsGroupRemovePermissionCommand extends CommandManager {

	public PermissionsGroupRemovePermissionCommand() {
		super("removepermission","removerpermissao");
		setUsage("/permissions group addpermission <group> <permission>");
		setDescription("Remover uma permissão do grupo");
	}
	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {

		PermissionsManager manager = EduPermissions.getInstance().getManager();
		if (args.length < 4) {
			sendUsage(sender);
		} else {
			String nome = args[2];
			PermissionsGroup group = manager.getGroup(nome);
			String perm = args[3];
			if (group != null) {
				group.getPermissions().remove(perm);
				sender.sendMessage(EduPermissions.getInstance().message("group-remove-permission")
						.replace("$group", "" + nome).replace("$permission", perm));
			} else {
				sender.sendMessage(
						EduPermissions.getInstance().message("group-not-exists").replace("$group", "" + nome));
			}
//			
		}

		return true;
	}

}

File: src/main/java/net/eduard/permissions/command/PermissionsGroupRemovePermissionCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command
import net.eduard.api.lib.command.Sender
import net.eduard.permissions.api.PermissionsAPI
import net.eduard.permissions.core.PermsMessages

class PermissionsGroupRemovePermissionCommand :

    Command("removepermission", "removerpermissao") {

    override fun onCommand(sender: Sender, args: List<String>) {
        val manager = PermissionsAPI.instance
        if (args.size < 4) {
            sendUsage(sender)
            return
        }
        val nome = args[2]
        val group = manager.getGroup(nome)
        val perm = args[3]
        if (group != null) {
            group.removePermission(perm)
            sender.sendMessage(
                PermsMessages.message("group-remove-permission")
                    .replace("\$group", "" + nome)
                    .replace("\$permission", perm)
            )
            return
        }
        sender.sendMessage(
            PermsMessages.message("group-not-exists")
                .replace("\$group", "" + nome)
        )

    }


    init {
        usage = "/permissions group removepermission <group> <permission>"
        description = "Remover uma permissão do grupo"
    }
}